tests/utils/lambda-helpers.mjs

```diff
@@ -0,0 +1,194 @@
+// @ts-check
+
+// this is not TS file because it's used both directly inside test process
+// as well as child process that lacks TS on-the-fly transpilation
+
+import { join } from 'node:path'
+import { BLOB_TOKEN } from './constants.mjs'
+import { execute as untypedExecute } from 'lambda-local'
+
+const SERVER_HANDLER_NAME = '___netlify-server-handler'
```

Comment:
this is copied from https://github.com/opennextjs/opennextjs-netlify/blob/e96cd1831d0a6b0f6429e08a77127a48467de56a/src/build/plugin-context.ts#L22 (for the serverless invocation in same process it was imported from there, but it was previously already copied for sandboxed invocation because it can't import TS, so not fully new that this is duplicated)